Documentation/platforms/aws.md

```diff
@@ -0,0 +1,77 @@
+## Introduction
+
+This guide assumes operational knowledge of Amazon Web Services (AWS), specifically Amazon Elastic Compute Cloud (EC2). This guide provides an introduction to design considerations when designing an etcd deployment on AWS EC2 and how AWS specific features may be utilized in that context.
+
+## Capacity planning
+
+As a critical building block for distributed systems it is crucial to perform adequate capacity planning in order to support the intended cluster workload. As a highly available and strongly consistent data store increasing the number of nodes in an etcd cluster will generally affect performance adversely. This makes sense intuitively, as more nodes means more members for the leader to coordinate state across. The most direct way to increase throughput and decrease latency of an etcd cluster is allocate more disk I/O, network I/O, CPU and memory to cluster members. In the event it is impossible to temporarily divert incoming requests to the cluster, scaling the EC2 instances which comprise the etcd cluster members one at a time may improve performance. It is, however, best to avoid bottlenecks through capacity planning.
+
+The etcd team has produced a [hardware recommendation guide](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/hardware.md) which is very useful for “ballparking” how many nodes and what instance type are necessary for a cluster.
```

Comment:
replace https://github.com/coreos/etcd/blob/master/Documentation/op-guide/hardware.md with relative link ../op-guide/hardware.md

otherwise coreos site doc machinery gets upset